bots/bridge-tester/test/integration/redeem.test.ts

```diff
@@ -30,7 +30,7 @@ async function produceBlocks(
   }
 }
 
-describe("redeem", () => {
+describe.skip("redeem", () => {
```

Comment:
Same as before: If the `skip` was added because the test needs fixing, maybe add a comment with an issue link?